src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -448,7 +453,7 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
 
             try
             {
-                await ExecuteWithWatchersAsync(instance, parameterHelper, trace, logger, functionCancellationTokenSource);
+                await ExecuteWithWatchersAsync(instance, parameterHelper, trace, logger, functionCancellationTokenSource, _jobHost, _config);
```

Comment:
The one issue with changing all of these to instance is the InvokeAsync method. Other tests and functions are expecting InvokeAsync to be static.